jekyll/_cci2/deployment-integrations.md

```diff
@@ -16,46 +14,60 @@ CircleCI can be configured to deploy to virtually any service. This document pro
 ## Overview
 {:.no_toc}
 
-To deploy your application, add a [job]({{ site.baseurl }}/2.0/jobs-steps/#jobs-overview) to `.circleci/config.yml`. If your deploy job uses any output from previous jobs, you can share that data by [using workspaces]({{ site.baseurl }}/2.0/workflows/#using-workspaces-to-share-data-among-jobs). You will also have to set any necessary secrets on the **Project Settings > Environment Variables** page of the CircleCI application.
```

Comment:
The original ticket added this line about workspaces, so I'm not sure why you are removing it in this revision? I would add this back.

Reply:
Ah, good catch. I think I meant to add it _after_ the configuration example.